### DISCUSSION TITLE
MilvusHybridIndexDemo.ipynb is incomplete?

### DISCUSSION THREAD

**Discussion by Danana1** [run-llama/llama_index#14042] [Q&A]
Hi, 
I was looking at the [https://github.com/run-llama/llama_index/blob/main/docs/docs/examples/vector_stores/MilvusHybridIndexDemo.ipynb ]  example.
It looks as if it is incomplete, or perhaps I missed something.
There is no DB connection (URI = http://localhost:19530) and no collection name.
and when I run it I get
 pymilvus.exceptions.MilvusException: <MilvusException: (code=0, message=104 data type not supported)>
 @wxywb FYI

**Comment by wxywb** [comment]
@Danana1 Yes, you need to start Milvus 2.4 before you connect.
Milvus 2.4 is an independent service that needs to be run in Docker by yourself [check this](https://milvus.io/docs/install_standalone-docker-compose.md). When you start the Milvus service, LlamaIndex can connect using this URI (http://localhost:19530).

Recently, we are offering a more convenient option called Milvus Lite, which doesn't require you to start an extra service; it is just a Python library. However, it still lacks some functionality, like hybrid retrieval. Please stay tuned if you are interested.

**Comment by Danana1** [comment]
Thank you. Indeed I have an older version of Milvus. Will try to upgrade
